### PR TITLE
Add more doc for the GetLegendGraphics request

### PR DESCRIPTION
**Pull request by pblottiere**

### CODE REVIEW

**Comment by pblottiere** [comment]
I rebased this PR with master and updated images with transparent background :).

File: docs/server_manual/services/wms.rst [hunk 1]
```diff
@@ -826,63 +825,729 @@ Available values are (not case sensitive):
...
+
+
+.. _`wms-getlegendgraphics-srcwidth`:
+
+SRCWIDTH
+^^^^^^^^
+
+This parameter may de defined when the ``RULE`` parameter is set. In this case,
+the ``SRCWIDTH`` value is forwarded to the underlying ``GetMap`` request as the
+``WIDTH`` parameter while the ``WIDTH`` parameter of ``GetegendGraphics`` is
```
Comment (DelazJ) [inline docs/server_manual/services/wms.rst:1005]:
```suggestion
``WIDTH`` parameter while the ``WIDTH`` parameter of ``GetLegendGraphics`` is
```

File: docs/server_manual/services/wms.rst [hunk 2]
```diff
@@ -826,63 +825,729 @@ Available values are (not case sensitive):
...
+
+
+.. _`wms-getlegendgraphics-srcheight`:
+
+SRCHEIGHT
+^^^^^^^^^
+
+This parameter may de defined when the ``RULE`` parameter is set. In this case,
+the ``SRCHEIGHT`` value is forwarded to the underlying ``GetMap`` request as
+the ``HEIGHT`` parameter while the ``HEIGHT`` parameter of ``GetegendGraphics``
```
Comment (DelazJ) [inline docs/server_manual/services/wms.rst:1016]:
```suggestion
the ``HEIGHT`` parameter while the ``HEIGHT`` parameter of ``GetLegendGraphics``
```

**Review by DelazJ — COMMENTED** [review]
Quick review

File: docs/server_manual/services/wms.rst [hunk 3]
```diff
@@ -367,9 +367,8 @@ Available values are:
 * ``image/png; mode=1bit``
 * ``image/png; mode=8bit``
 * ``image/png; mode=16bit``
-* ``application/dxf``
-  Only layers that have read access in the WFS service are exported in
-  the DXF format.
+* ``application/dxf``: only layers that have read access in the WFS service are
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wms.rst:372]: I think it's correct as it is formulated at the moment.

**Comment by pblottiere** [comment]
H @DelazJ,

I noticed your reticence about adding images in the documentation so I removed them to move forward. 

TBH I still think it's a shame because the way I see it, the training manual is for training (lessons, real case examples, ...) but not for understanding concepts. For example I think the [Catalog](https://docs.qgis.org/3.22/en/docs/server_manual/catalog.html) chapter would be much more difficult to understand without screenshots... But anyway, we can talk about it later :).

Anyway, thanks a lot for you review! Let me know if I have to update this PR somehow :).

File: docs/server_manual/services/wms.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/server_manual/services/wms.rst:372]: Ah... OK. Just surprised to see mention of wfs in a wms doc. I thought it was a typo.

**Comment by DelazJ** [comment]
> I noticed your reticence about adding images in the documentation

Euh... You sure 🤔 ? Mind pointing me to where i said that because there's probably some misinterpretation? I do not remember I said that. I'm OK as long as they help and do not need maintenance hassle.
And again, this is a community project and I don't claim and don't want to be the only one who decides what is best or no.

**Comment by DelazJ** [comment]
Reading again the most recent discussion on the topic (#7220) I think I found the problem: https://github.com/qgis/QGIS-Documentation/pull/7220#discussion_r771506642. I badly quoted your message and wrote a not accurate reply. What was unnecessary, is the script. I was suggesting to preserve your time from developing a script of maintenance, not from generating screen shots.

**Comment by DelazJ** [comment]
> Anyway, thanks a lot for you review! Let me know if I have to update this PR somehow :)

```git reset --hard HEAD^ && git push -f```?

**Comment by pblottiere** [comment]
Hi @DelazJ :),

> Euh... You sure ￼ ? Mind pointing me to where i said that because there's probably some misinterpretation? I do not remember I said that. I'm OK as long as they help and do not need maintenance hassle.

Yes it was indeed this discussion: https://github.com/qgis/QGIS-Documentation/pull/7220#discussion_r771443946.

> And again, this is a community project and I don't claim and don't want to be the only one who decides what is best or no.

Indeed but I think your opinion is much more valuable than mine on the documentation repository :sweat_smile:.  I trust your judgement so if you're reluctant, so am I ^^. 

Moreover, the more I read the QGIS Server documentation, the more I think that the line between the QGIS Server documentation and the training manual is unclear. For example, what do you think about the [Containerized deployment](https://docs.qgis.org/3.22/en/docs/server_manual/containerized_deployment.html) chapter?

> `git reset --hard HEAD^ && git push -f`?

I's fine for me :).

And again, thanks again for all your work on the documentation,it's awesome :+1:.

**Comment by pblottiere** [comment]
I think this PR is ready to be merged :).

**Comment by DelazJ** [comment]
I moved the containerized deployment question to #7244 to avoid it blocking this PR and have it addressed later by skilled people.

**Pull request merged**